client/jetpack-connect/test/authorize.js

```diff
@@ -76,9 +100,9 @@ jest.mock( '@automattic/calypso-config', () => {
 
 describe( 'JetpackAuthorize', () => {
 	test( 'renders as expected', () => {
-		const wrapper = shallow( <JetpackAuthorize { ...DEFAULT_PROPS } /> );
+		const { container } = renderWithRedux( <JetpackAuthorize { ...DEFAULT_PROPS } /> );
 
-		expect( wrapper ).toMatchSnapshot();
+		expect( container ).toMatchSnapshot();
 	} );
 
 	describe( 'isSso', () => {
```

Comment:
Not necessary for this PR, as this is pre-existing, but if this can be tested without rendering, it's usually a good candidate for extracting to a separate utility function.